makeMALI

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
@tongzhangice , please remove this file.

Reply:
Are you looking at an old branch Matt?

Reply:
I was reviewing the newest commit you added one at a time, and I didn't realize that you undid some of those changes with later commits.  I made it through all of it and it all look ok.  I added one more commit with a few corrections to typos.  I'm testing with our full regression suite now.

Reply:
Thanks. I should've let you know about that!